### ISSUE TITLE
Unable to configure P2P_ANNOUNCE_ADDRESSES for ipv6

### ISSUE THREAD

**Issue by TanLocc** [oceanprotocol/ocean-node#751]
My network provider uses a dynamic IP for all devices and assigns ipv6 identifiers to the devices.

And the problem is that my PC will only use ipv6 to configure P2P_ANNOUNCE_ADDRESSES, but the ipv6 provided to the device is dynamic, so my PC device keeps changing ipv6 address, so I have difficulty in configuring P2P_ANNOUNCE_ADDRESSES.

I hope that in the future, checking nodes will not need to configure P2P_ANNOUNCE_ADDRESSES, or someone can give me a solution to solve this problem.

**Comment by alexcos20** [comment]
Can you try to remove P2P_ANNOUNCE_ADDRESSES?

You can always check how your node sees the network (including announced & observed addresses) with something like:

```
curl http://localhost:8000/getP2pNetworkStats
```
but you would have to enable it by using export P2P_ENABLE_NETWORK_STATS=true

**Comment by TanLocc** [comment]
> Can you try to remove P2P_ANNOUNCE_ADDRESSES?
> 
> You can always check how your node sees the network (including announced & observed addresses) with something like:
> 
> ```
> curl http://localhost:8000/getP2pNetworkStats
> ```
> 
> but you would have to enable it by using export P2P_ENABLE_NETWORK_STATS=true
Tôi đã remove P2P_ANNOUNCE_ADDRESSES, và thêm P2P_ENABLE_NETWORK_STATS=true trong docker-compose.yml, và đầy là đầu ra của : 
`curl http://localhost:8000/getP2pNetworkStats`
![image](https://github.com/user-attachments/assets/3e708090-631a-40dd-9962-4b20e9a09686)
But does removing P2P_ANNOUNCE_ADDRESSES affect node performance and rewards?

**Comment by alexcos20** [comment]
looks like your node listens only on ip4, and not ipv6
Enable ipv6 please, and #794 might help you

**Comment by TanLocc** [comment]
@alexcos20  Yes I have to wait for the commit to be merged and build to the latest image to be able to use ipv6
